src/server.ts

```diff
@@ -1,8 +1,73 @@
 import express, { Request, Response } from 'express';
 
+import { Prisma, PrismaClient, User } from '@prisma/client';
```

Comment:
the prisma/client is a unique-to-your-schema artifact that is generated when you run `npx prisma generate`.

To that end, I've included it in the postinstall script so that when we deploy the server, it has the `@prisma/client` artifact before builds happen.